package.json

```diff
@@ -49,7 +49,7 @@
         "@fastify/csrf-protection": "^4.0.1",
         "@fastify/helmet": "8.1.0",
         "@fastify/static": "^5.0.2",
-        "@flowforge/forge-ui-components": "^0.3.0",
+        "@flowforge/forge-ui-components": "0.3.1",
```

Comment:
```suggestion
        "@flowforge/forge-ui-components": "^0.3.1",
```

Reply:
agreed, certainly within ff cloud our approach is to use small, medium and not directly quote the underlying resources.
So for this PR I would favour just showing the name of the stack for now, we have the description field within the type where more information can be provided and at present all our stacks within a project type would have the same capacity anyway.

There is some UX improvements to consider around how we present the stack, as just the unique name field isn't the most user friendly, but I would favor adding a description box like the project type and allow free text here.

Reply:
In that case, I vote a hold on this for the 0.8 Release to make sure we get this right.